akka-docs/src/main/paradox/scala/cluster-usage.md

```diff
@@ -592,16 +644,35 @@ the average number of characters per word when all results have been collected.
 
 Messages:
 
-@@snip [StatsMessages.scala]($akka$/akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsMessages.scala) { #messages }
+Scala
+:  @@snip [StatsMessages.scala]($akka$/akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsMessages.scala) { #messages }
+
+Java
+:  @@snip [StatsMessages.java]($code$/java/jdocs/cluster/StatsMessages.java) { #messages }
 
 The worker that counts number of characters in each word:
 
-@@snip [StatsWorker.scala]($akka$/akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsWorker.scala) { #worker }
+Scala
+:  @@snip [StatsWorker.scala]($akka$/akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsWorker.scala) { #worker }
+
+Java
+:  @@snip [StatsWorker.java]($code$/java/jdocs/cluster/StatsWorker.java) { #worker }
 
 The service that receives text from users and splits it up into words, delegates to workers and aggregates:
 
+@@@ div { .group-scala }
+
 @@snip [StatsService.scala]($akka$/akka-cluster-metrics/src/multi-jvm/scala/akka/cluster/metrics/sample/StatsService.scala) { #service }
 
+@@@
+
+@@@ div { .group-java }
+
+@@snip [StatsService.java]($code$/java/jdocs/cluster/StatsService.java) { #service }
+@@snip [StatsAggregator.java]($code$/java/jdocs/cluster/StatsAggregator.java) { #aggregator }
+
+@@@
+
```

Comment:
Using `@@@ div { .group-scala }` and `@@@ div { .group-java }` because there are two `@@snip` directives for Java in **two different .java files**. 

(i.e.) below didn't work:
```
Scala
:  @@snip ...

Java
:  @@snip...
```